docs/requirements-specification.md

```diff
@@ -57,4 +57,49 @@ _This content item details the use case overview of the system vision in its use
 
 ### UC5: Find expert in a field
 
-### UC6: Organizing capabilities
\ No newline at end of file
+### UC6: Organizing capabilities
+Only Create (C) use cases are explicitly defined here. Update (U) and Delete (D) need still to be defined and shall not contradict/invalidate Create use cases.
+
+| UC6a | Create capability group |
+|---|---|
+| Actors | Admin |If the capability group name exists already, the system prevents the Admin from creating the capability group. |
+| Pre-conditions | The admin is authenticated and has a valid, digital evidence. |
+| End conditions (success) | The capability group is created. |
+| End conditions (failure | The capability group is not created. |
+| Steps | 1. If an Admin presses "Create capability group", the system opens a dialog window with information fields required for a new capability group. |
+| | 2. The Admin enters the information for a new capability group: name, description, and importance (I forgot what we thought about this attribute). |
```

Comment:
I would not formulate functional requirements (i.e., steps of a use case) in a way that imposes on what a user does: we cannot specify what the admin does; we can only specify how the system reacts *if* the admin does something. Hence, I would merge this step with the previous one: "... the system opens a dialog window with information fields required for a new capability group, **containing the following fields: ...**".

Reply:
Ok, merged in all three UCs.